common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/boost_polygon_utils.hpp

```diff
@@ -27,13 +29,14 @@
 
 namespace tier4_autoware_utils
 {
-
 bool isClockwise(const Polygon2d & polygon);
 Polygon2d inverseClockwise(const Polygon2d & polygon);
 geometry_msgs::msg::Polygon rotatePolygon(
   const geometry_msgs::msg::Polygon & polygon, const double & angle);
 Polygon2d toPolygon2d(
   const geometry_msgs::msg::Pose & pose, const autoware_auto_perception_msgs::msg::Shape & shape);
+Polygon2d toPolygon2d(const autoware_auto_perception_msgs::msg::DetectedObject & object);
```

Comment:
@scepter914 Do you have any reason to add toPolygon of perception in tier4_autoware_utils instead of perception_utils?